Sources/AppStoreConnectCLI/Commands/Users/ListUsersCommand.swift

```diff
@@ -69,7 +69,7 @@ public struct ListUsersCommand: CommonParsableCommand {
             include: includeVisibleApps
                 ? [ListUsers.Include.visibleApps]
                 : nil,
-            limit: nil, // Limit of visibleApps if included, not limit of users
+            limit: limit.map { [ListUsers.Limit.visibleApps($0)] } ?? [], // Limit of visibleApps if included, not limit of users
```

Comment:
I think we can remove this comment since what it is doing is described by the enum case. As well as will be described by the new variable name.